submissions/Complexity Corner - Speakers/Complexity Corner - Speakers.md

```diff
@@ -0,0 +1,23 @@
+### Complexity Corner: Speakers <author>The Kool</author>
+
+Welcome back to Complexity Corner, where we look over the less used or more complex mechanics of the game to give an introduction to the unfamiliar.  In this article, we'll ask three questions: 'What is it?', 'What results can it give?', and 'Where should it be used?'
+
+What is it?
+
+The [Programmable Speaker](https://wiki.factorio.com/Programmable_speaker) is a 'building' that takes up only one tile.  When connected to a circuit, it can make sounds that you'll hear in game based on the signals on that circuit.  You can adjust the volume, pick a specific sound for it to play, tell it to be audible no matter how far away you are, and even show a custom alert on the map.  The speaker is unlocked with the Circuits technology, which is so early in the tech tree that it could be your first green tech.
+
+What results can it give?
+
+Sounds!  Okay, that one was kind of a gimme.  While speakers are best known for [the musical contraptions](https://miditorio.com/) people build with them, they hold a practical use as well.  If you hook a speaker up to your factory and set global playback and an alert icon, you can design custom alerts for just about anything.  You even get to give it a custom icon and title, because you'll absolutely forget what the blaring klaxon is for.  This can help you catch problems before they become problems, like blackout spirals in your power grid.
+
+Where should it be used?
+
+Speaking of blackout spirals, try placing a speaker next to the coal belt leading to your boilers and hooking it to the belt with a wire.  A full belt holds 8 items, so if you set the speaker to trigger on less than 8 coal, you'll get an alarm if any gaps show up in your coal belt.  Set this alarm to global playback and giving it a name like "Check on your coal mining!" and you'll get the reminder even if you're halfway across the factory.
```

Comment:
> Speaking of blackout spirals

I think it's called the Coal Death Spiral, but I'm not sure. Maybe you could also very quickly mention what it is?

Reply:
I cut that because I imagined most people have experienced it.  Thinking back, I've seen that explained on reddit several times, so I'll add it back in